FirebaseSessions/Tests/Unit/InitiatorTests.swift

```diff
@@ -31,7 +32,10 @@ class InitiatorTests: XCTestCase {
   func test_appForegrounded_initiatesNewSession() throws {
```

Comment:
Can we create a testcase where the timeout interval is used to generate a new session (the timeout interval can be set to a low value, say 1 second)?